libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -234,6 +235,7 @@ def stream(
                 for chunk in self._stream(
                     messages, stop=stop, run_manager=run_manager, **kwargs
                 ):
+                    _cp_gen_info_to_msg(chunk)
```

Comment:
I would find it more intuiitve if we didn't mutate in thefunction and instead did

```
chunk.message.additional_kwargs = _merge_gen_info(chunk)
```
or something (instead of mutating)

Reply:
tend to agree